docs/module/gateway.md

```diff
@@ -0,0 +1,63 @@
+# First-Hop Gateway Configuration Module
+
+First-hop Gateway configuration module implements mechanisms used to implement a shared router IPv4 address on a stub access network.
+
+The module supports statically configured anycast gateway IPv4 address and VRRPv3 for IPv4 and IPv6.
+
+The module is supported on these platforms:
+
+| Operating system      | Anycast | VRRPv3 | VRRPv3<br>IPv6 |
+| --------------------- | :-: | :-: | :-: |
+| Arista EOS            | ✅  | ✅  | ✅  |
+| Cisco CSR 1000v       | ✅  | ✅  | ✅  |
+| Cisco IOSv            | ✅  | ✅  | ✅  |
+| Cumulus Linux         | ✅  | ✅  | ✅  |
+
+## Global Parameters
+
+The module supports the following global parameters: 
+
+* **gateway.protocol** (default: *anycast*) -- the first-hop gateway resolution protocol. The only supported value is currently *anycast*
+* **gateway.id** (default: -1) -- the IP address within the subnet used for the gateway IP address
+
+## Link Parameters
+
+Gateway configuration module is enabled on all links that have **gateway** attribute set to *True* or to a dictionary of valid parameters. You can change most global parameters on per-link basis.
+
+## Anycast Gateway
+
+The *gateway* configuration module supports IPv4 anycast gateways -- MAC and IPv4 addresses shared between multiple nodes attached to the same segment.
+
+### Implementation Notes
+
+* The default *netlab* shared MAC address is 0200.cafe.00ff.
+* Vendors use different names for anycast gateways: VARP (Arista), VRR (Cumulus)
+* Arista EOS uses the same shared MAC address on all interfaces. Do not set link-level **gateway.anycast.mac** parameter in topologies using Arista EOS.
+* Arista vEOS cannot use VRRP MAC address as a shared MAC address (default recommended by Cumulus Linux)
+
+### Anycast Parameters
+
+Anycast implementation of shared first-hop IPv4 address supports these parameters that can be specified globally or on individual links or interfaces.
+
+* **gateway.anycast.unicast** (default: True) -- configure node-specific unicast IP addresses together with anycast IP address.
+* **gateway.anycast.mac** -- Static MAC address used for the anycast IP address
```

Comment:
SROS by default generates a different virtual mac address for ipv4 versus ipv6 (derived from vrrp.group)

Reply:
In most (static) anycast implementations you have to specify the MAC address and while CL recommends using a MAC address from the VRRP range, Arista refuses to use that. That's why I chose another MAC address as the default value.

Also, most static anycast implementations support only IPv4. You could configure IPv6 as well if you wish; the IPv6 address is in the data model.